### PR TITLE
List games

### PR DESCRIPTION
**Pull request by Collin-Moore**
GetGames now only requests the data it needs from redis and returns it as a map[string]interface{} to my handlers

### CODE REVIEW

File: service/Handlers.go [hunk 1]
```diff
@@ -20,6 +20,16 @@ func HeartBeat(w http.ResponseWriter, r *http.Request) {
 // AdminPasswordCheck determines if the client can access the admin pages
 func AdminPasswordCheck(w http.ResponseWriter, r *http.Request) {
 	// TODO: Remember to defer some recovery code here
```
Comment (ckeinsley) [inline service/Handlers.go]: Is this todo still todo?

File: service/Handlers.go [hunk 2]
```diff
@@ -63,7 +73,26 @@ func CreateGame(w http.ResponseWriter, r *http.Request) {
 
 // DeleteGame will delete the game with the matching id
 func DeleteGame(w http.ResponseWriter, r *http.Request) {
+	// TODO consider refactoring to use recover package
```
Comment (ckeinsley) [inline service/Handlers.go:75]: are we still considering this?

File: service/Handlers.go [hunk 3]
```diff
@@ -97,65 +126,36 @@ func ListGames(w http.ResponseWriter, r *http.Request) {
...
-	}
-}
-
-// Store handles POST requests to /store
-// This converts the object to a checkoff model and
-// sends it to long term storage
-/*func Store(w http.ResponseWriter, r *http.Request) {
+// GameInfo is like ListGames but for a single game id in the query params
+func GameInfo(w http.ResponseWriter, r *http.Request) {
+	// TODO consider refactoring to use recover package
```
Comment (ckeinsley) [inline service/Handlers.go:140]: Still considering?

**Review by ckeinsley — CHANGES_REQUESTED** [review]
See in line comments

File: service/Handlers.go (hunk 1 above)
Comment (Collin-Moore) [reply service/Handlers.go]: Nope, fixed I think

**Review by ckeinsley — APPROVED** [review]
Yay fewer comments!

File: service/Handlers.go (hunk 2 above)
Comment (Collin-Moore) [reply service/Handlers.go:75]: for now yes

File: service/Handlers.go (hunk 3 above)
Comment (Collin-Moore) [reply service/Handlers.go:140]: for now yes

**Pull request merged**